include/aws/mqtt/client.h

```diff
@@ -228,7 +228,11 @@ int aws_mqtt_client_connection_set_connection_interruption_handlers(
  *                                          This is copied into the connection
  *                                          Pass NULL to connect without TLS (NOT RECOMMENDED)
  * \param[in] clean_session             True to discard all server session data and start fresh
- * \param[in] keep_alive_time           The keep alive value to place in the CONNECT PACKET
+ * \param[in] keep_alive_time           The keep alive value to place in the CONNECT PACKET, a PING will automatically
+ *                                       be sent at this interval as well. If you specify 0, defaults will be used
+ *                                       and a ping will be sent every 60s.
+ * \param[in] request_timeout_ms        Amount of time before a message is considered timed out. Default is 3s.
+ *                                      Specify 0 for defaults.
```

Comment:
This sounds like it applies to normal PUB/SUB MQTT messages, which shouldn't have timeouts in this async API. If they're QoS 1, then the client should just keep them queued until a reconnect is finally established.

This is only supposed to apply to the keepalive ping, right?

Reply:
i'll update the comment.